src/bip39.cpp

```diff
@@ -31,19 +31,19 @@
 
 #include <openssl/evp.h>
 
-SecureString mnemonic_generate(int strength)
+SecureString CMnemonic::Generate(int strength)
 {
     if (strength % 32 || strength < 128 || strength > 256) {
         return SecureString();
     }
     uint8_t data[32];
     GetRandBytes(data, 32);
-    SecureString mnemonic = mnemonic_from_data(data, strength / 8);
+    SecureString mnemonic = FromData(data, strength / 8);
     memory_cleanse(data, sizeof(data));
```

Comment:
memory_cleanse doesn't prevent data from being swapped. Would need to use LockObject followed by UnlockObject.  It would be simpler to just replace the local buffer by a SecureVector.  None of this code should be very performance critical so the small cost of memory allocation should be negligible.

Also most newer bitcoin code seems to be moving away from the use of LockObject to using the secure allocator.

Same comment applies to all functions in bip39 that use memory_cleanse.